compendium/ConfigurationAdmin/src/ConfigurationAdminImpl.hpp

```diff
@@ -269,7 +268,7 @@ namespace cppmicroservices
                 cppmicroservices::service::cm::ManagedServiceFactory,
                 TrackedServiceWrapper<cppmicroservices::service::cm::ManagedServiceFactory>>
                 managedServiceFactoryTracker;
-            std::mt19937 randomGenerator;
+            thread_local static std::mt19937 randomGenerator;
```

Comment:
this doesn't seem right. Why not move `randomGenerator` from being a member variable to a `static thread_local` scoped variable within `ConfigurationAdminImpl::RandomInstanceName()`?

Reply:
Will fix